src/Illuminate/Console/Scheduling/ManagesFrequencies.php

```diff
@@ -121,6 +121,21 @@ public function everyThirtyMinutes()
         return $this->spliceIntoPosition(1, '0,30');
     }
 
+    /**
+     * Schedule the event to run every specified minutes.
+     *
+     * @param  int $minutes
+     * @return $this
+     */
+    public function everyFewMinutes(int $minutes)
```

Comment:
None of the other methods in this class utilize type hints.

```suggestion
    public function everyFewMinutes($minutes)
```